### ISSUE TITLE
Discuss: Hook unit tests to cloud CI server, such as Travis, CircleCI, or Jenkins.

### ISSUE THREAD

**Issue by dariusstrasel** [mechamecho/DataDrivenTesting#16]

**Comment by dariusstrasel** [comment]
I've deployed a Jenkins server to Azure. URL is:
http://jenkins-alk.eastus.cloudapp.azure.com

The front-end is disabled, as of this writing, but I will be turning it on for us to mess with it.
@mechamecho @Neous1

**Comment by mechamecho** [comment]
Awesome, great work!

**Comment by dariusstrasel** [comment]
Jenkins URL:
jank-jenkins.eastus.cloudapp.azure.com

Instructions to come.

**Comment by mechamecho** [comment]
Which name and password are you referring to? Is it one that I create for myself?

**Comment by dariusstrasel** [comment]
Yes. Run the SSH command and it will guide you through the setup. Let me
know if you need help.

**Comment by dariusstrasel** [comment]
Azure is killing me, lol. I've add to rebuild the server like 5 times now.

**Comment by mechamecho** [comment]
That sounds bad lol
